app/src/androidTest/java/org/simple/clinic/drugs/search/DrugRepositoryAndroidTest.kt

```diff
@@ -85,6 +114,6 @@ class DrugRepositoryAndroidTest {
         .blockingFirst()
 
     // then
-    assertThat(expectedSearchResults).containsExactly(amlodipine10, amlodipine20).inOrder()
+    assertThat(expectedSearchResults).containsExactly(amlodipine10).inOrder()
```

Comment:
Can we also rename the `DrugRepository#search` method to make it explicit that this excludes drugs found in protocols?